.chloggen/TEMPLATE.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component:
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
```

Comment:
Should the PR be always included as well so it's linked to the entry, or do we expect readers to go to the issue and then from there to the referenced PR?

Reply:
> or do we expect readers to go to the issue and then from there to the referenced PR

I think that should be the default experience, yes. If an issue does not exist then a PR can be used. 

I much prefer this way because it fixes what we have today -  one has to open the PR then grab the number to add to the changelog.